astroquery/vo_service_discovery/tests/shared_registry.py

```diff
@@ -0,0 +1,169 @@
+import os
+from astropy.table import Table
+from astroquery.vo_service_discovery import Registry
+
+
+class Helpers(object):
+    ""
+
+    DATA_FILES = {}
+
+    def table2ecsv(self, current, filepath):
+        """Dump a table and its meta data to ECSV"""
+        from astropy.io import ascii as ap_ascii
+        try:
+            ap_ascii.write(current, filepath, format='ecsv', overwrite=True)
+        except Exception as e:
+            raise e
+
+    def ecsv2table(self, filepath):
+        """
+        """
+        try:
+            with open(filepath, 'r') as f:
+                table = Table.read(f.read(), format='ascii.ecsv')
```

Comment:
`Table` should be able to deal with a file name, doesn't need an open file

Reply:
Yes, that looks much cleaner.  Thanks for the review!